usdr-gost.code-workspace

```diff
@@ -27,5 +27,12 @@
 		// to manage linting/fixing, but users can have other settings via "formatOnSave".
 		// If both are on, the editor can "fix" things that are immediately flagged as lint failures.
 		"editor.formatOnSave": false,
+		// Uncomment the vitest.shellType setting if the "Testing" panel is empty and/or 
+		//   you see vitest extension errors in the output. The extension
+		//   sometimes cannot find the right version of Node unless it using a terminal shell
+		//   to spawn the test running process.
+		//  see: https://github.com/vitest-dev/vscode/issues/486 
+		//         https://github.com/vitest-dev/vscode/pull/490
+		//  "vitest.shellType": "terminal",
```

Comment:
@phm200 Are there any downsides to this being uncommented by default?

Reply:
good question. I'm not entirely sure. Based on the GH activity around this feature, this should still work the same for folks who were fine with the default "child process" setting. If you or someone else whose vitest plugin is working normally can try it out and confirm it works, then I can uncomment it for all

to your point, if we cannot uncomment it for all, probably better for me to just add a note to the readme and people can make that change in their own settings json and not have to make an un-commitable change to the .code-workspace every time

Reply:
Just checked, and I seem to be able to run tests using the vitest plugin successfully, regardless of whether this line is (un)commented, as well as when explicitly set to `child_process`.

In case you think it matters, I'll add that my dev environment are fairly ephemeral, so the only Node I have is what's installed by nvm (there's no global/system default), and I only typically have the version specified in this repo's `.nvmrc` installed.

Reply:
will do, uncommented in [566a194](https://github.com/usdigitalresponse/usdr-gost/pull/3605/commits/566a1946fafa28669bac5687cbb69637008b8064)